pkgs/by-name/de/demucs-cpp/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  blas,
+}:
+stdenv.mkDerivation rec {
+  pname = "demucs-cpp";
+  version = "0.0.4-alpha";
+  src = fetchFromGitHub {
+    fetchSubmodules = true;
+    owner = "sevagh";
+    repo = "demucs.cpp";
+    rev = "v${version}";
```

Comment:
`rev = "refs/tags/v${version}"` is safer than `rev = "v${version}` if there is ever a matching branch name